test/mz-debug/mzcompose.py

```diff
@@ -48,27 +47,8 @@ class TestCase:
 
 
 def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
-    c.up("materialized")
-
-    mz_debug = c.compose["services"]["mz-debug"]
-    subprocess.run(
-        ["docker", "pull", mz_debug["image"]],
-        check=True,
-        capture_output=True,
-    )
-    container_id = subprocess.check_output(
-        ["docker", "create", mz_debug["image"]], text=True
-    ).strip()
-    subprocess.run(
-        [
-            "docker",
-            "cp",
-            f"{container_id}:/usr/local/bin/mz-debug",
-            ".",
-        ],
-        check=True,
-    )
-
+    c.up("materialized", {"name": "mz-debug", "persistent": True})
+    c.invoke("cp", "mz-debug:/usr/local/bin/mz-debug", ".")
```

Comment:
It feels a bit weird to be pulling this out of the container. Why is it in the materialized image in the first place?